enhancements/cluster-logging/forward_to_cloudwatch.md

```diff
@@ -0,0 +1,198 @@
+---
+title: forward_to_cloudwatch
+authors:
+  - "@alanconway"
+reviewers:
+  - "@jcantrill"
+  - "@jeremyeder"
+approvers:
+creation-date: 2020-12-17
+last-updated: 2020-12-17
+status: implementable
+see-also:
+superseded-by:
+---
+
+# Forward to Cloudwatch
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+[Amazon Cloudwatch][aws-cw] is a hosted monitoring and log storage service.
+This proposal extends the `ClusterLogForwarder` API with an output type for Cloudwatch.
+
+## Motivation
+
+Amazon Cloudwatch is a popular log store.
+We have requests from external and  Red Hat-internal customers to support it.
+
+### Goals
+
+Enable log forwarding to Cloudwatch.
+
+### Non-Goals
+
+Enable Cloudwatch metric collection.
+
+## Proposal
+
+New API fields in the `output.cloudwatch` section:
+
+- `region`: (string) AWS region name.
+- `retentionDays`: (number) Number of days to keep logs. Has a default.
+- `group`: (string, *nice-to-have*) Cloudwatch log group name.
+   Defaults to the log type (application, infrastructure or audit)
+- `groupLabel`: (string, *nice-to-have*) Kubernetes label to use as group name.
+  If the label is absent, use value of `group`.
+- `fetchIntervalSeconds`: (number, *nice-to-have*) interval to fetch logs in seconds.
```

Comment:
Why do we need to be able to fetch logs in order to push them to storage?

Reply:
We don't, my mistake.